akka-stream-tests-tck/src/test/scala/akka/stream/tck/Reproducer.scala

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.tck
+
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Flow
+import org.junit.Test
+import org.reactivestreams.Processor
+import org.scalatest.WordSpecLike
+
+class ReproducerTest extends AkkaIdentityProcessorVerification[Int] {
+
+  @Test
+  @throws[Throwable]
+  def reproduce(): Unit = {
+    var i = 0
+    while (i < 100000) {
+      System.out.println("Run " + i)
+      this.required_spec109_mustIssueOnSubscribeForNonNullSubscriber()
+      i += 1
+    }
+  }
```

Comment:
I'll remove this, but just to show that it passes the PR-validation (I got this to fail even locally before fix)

Reply:
Ok please remove then :)